source/getting-started/installation-raspberry-pi-image.markdown

```diff
@@ -39,13 +39,18 @@ Some extra tips:
  - Home Assistant is installed in a virtual Python environment at `src/homeassistant`
  - Home Assistant will be started as a service run by the user `homeassistant`
 
+Accessing Your Raspberry Pi via SSH:
+
+ - Connect using Terminal.app or iTerm.app on a Mac, Putty on Windows, or other Terminal application via entering `ssh pi@XXX.XXX.XXX.XXX` (where X = your Raspberry Pi's local IP address).
+ - Change the default password for the `pi` user by running `passwd` once connected via SSH.
+  
+
 {% comment %}
 
   TODO:
 
   Add instructions:
 
-   - How to login via shell (?) or at least mention how to work with a Pi
```

Comment:
This should probably be expanded on further. I'll merge your change in and use as a working base to work-on further.